### DISCUSSION TITLE
建议给pr也引进lable

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#1777] [提问] [answered]
给pr引进lable可以使其状态更清楚，比如已经提交了审阅人，就是`待处理`

**Comment by WForst-Breeze** [comment, marked as answer]
其实之前是有添加的，不过我想现在直接 requested a review 也就跟 `待处理` label 的效果差不多了（
![image](https://user-images.githubusercontent.com/110760354/235678991-bb5c8cd6-361b-4f1e-8c3d-a35cc147ac1e.png)
